### PR TITLE
Blog posts in pt

### PR DESCRIPTION
**Pull request by renatomello**

### CODE REVIEW

File: content/pt/about.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
-title: "About"
+title: "Sobre"
 subtitle: "About page"
-layout: "about"
+layout: "sobre"
```
Comment (deadit) [inline content/pt/about.md:4]: it is no need to change

File: content/pt/basics.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
-title: "Basics"
+title: "Fundamentos"
 subtitle: "Basics page"
-layout: "basics"
+layout: "fundamentos"
```
Comment (deadit) [inline content/pt/basics.md:4]: and this

File: content/pt/contribute.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
-title: "Contribute"
+title: "Contribua"
 subtitle: "Contribute page"
-layout: "contribute"
+layout: "contribua"
```
Comment (deadit) [inline content/pt/contribute.md:4]: also no need to change this

File: content/pt/documents.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ---
-title: "Documents"
+title: "Documentos"
 subtitle: "Documents page"
-layout: "documents"
+layout: "documentos"
```
Comment (deadit) [inline content/pt/documents.md:4]: and this

File: content/pt/faq.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
 ---
-title: "Faq"
+title: "Perguntas Frequentes"
 subtitle: "Frequently asked questions page"
-layout: "faq"
+layout: "perguntas_frequentes"
```
Comment (deadit) [inline content/pt/faq.md:4]: and this

File: content/pt/foundation.md [hunk 6]
```diff
@@ -1,7 +1,7 @@
 ---
-title: "Foundation"
+title: "Fundação"
 subtitle: "Foundation page"
-layout: "foundation"
+layout: "fundacao"
```
Comment (deadit) [inline content/pt/foundation.md:4]: and this

File: content/pt/hall_of_fame.md [hunk 7]
```diff
@@ -1,6 +1,6 @@
 ---
-title: "Hall of Fame"
+title: "Hall da Fama"
 subtitle: "Hall of Fame page"
-layout: "hall_of_fame"
+layout: "hall_da_fama"
```
Comment (deadit) [inline content/pt/hall_of_fame.md:4]: and this

File: content/pt/privacy.md [hunk 8]
```diff
@@ -1,7 +1,7 @@
 ---
-title: "Privacy"
+title: "Privacidade"
 subtitle: "Privacy page"
-layout: "privacy"
+layout: "privacidade"
```
Comment (deadit) [inline content/pt/privacy.md:4]: and this

File: content/pt/search.md [hunk 9]
```diff
@@ -1,6 +1,6 @@
 +++
-title = "Search"
-layout = "search"
+title = "Busca"
+layout = "busca"
```
Comment (deadit) [inline content/pt/search.md:3]: and this

File: content/pt/use_cases.md [hunk 10]
```diff
@@ -1,7 +1,7 @@
 ---
-title: "Use Cases"
+title: "Casos de Uso"
 subtitle: "Use Cases page"
-layout: "use_cases"
+layout: "casos_de_uso"
```
Comment (deadit) [inline content/pt/use_cases.md:4]: and this

**Comment by renatomello** [comment]
Fixed all of them

**Pull request merged**